docs/run-component-guide.md

```diff
@@ -1,24 +1,24 @@
-## Running the gem as a standalone app
+## Running the component guide as a standalone app
 
 If you want to work on this gem directly, without having to add it to another
-application, you can run it in the development vm or on your local machine with:
+application, you can run it on your local machine or using [govuk-docker](https://github.com/alphagov/govuk-docker). You'll need [Ruby](https://www.ruby-lang.org/en/downloads/) and [Node.js](https://nodejs.org/en/) installed to get this project running.
```

Comment:
I don't think you need Node.js on the local machine to get this running - I thought that Node is within a docker container?